english/friendly-person/friendly-person.md

```diff
@@ -22,4 +22,6 @@ Or [attempt to communicate via telepathy](telepathy/telepathy.md).
 
 You begin playing [charades](charades/charades.md).
 
-[Panic!!](panic/panic.md)
\ No newline at end of file
+[Panic!!](panic/panic.md) 
+
+or [have a Monty-Pythonesque experience](Monty-Python/Monthy-Python.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.